### DISCUSSION TITLE
trying to read voltage from the 12v battery pack, Hyundai Ioniq5

### DISCUSSION THREAD

**Discussion by tung256** [meatpiHQ/wican-fw#167] [Q&A]
in status, i see 14.6v.  good.
![image](https://github.com/user-attachments/assets/ccb33593-f66d-469e-877d-52fd99c8b173)

using Car Scanner app, i click on All Sensor and then i chose  OBD module voltage 
![image](https://github.com/user-attachments/assets/f6221e42-30b8-4ce0-8ac9-4273afa76de5)
![image](https://github.com/user-attachments/assets/6bcad74c-267c-4cae-a208-33a0c546f5c3)

in MQTT explorer, i see these updates:
![image](https://github.com/user-attachments/assets/743fe498-dd26-487a-a261-6dc0f0cb15cd)

what values should i put in here? 
![image](https://github.com/user-attachments/assets/cda24f6b-689e-459f-850e-17eb4168f81e)

this CSV file you sent over email does not make sense because i dont see CAN ID, Name, PID in your csv file.
please help. thanks!

**Comment by meatpiHQ** [comment]
@tung256 lets get some readings other than the battery voltage first as mentioned in #165 then I will help you set it up.
